src/codex/codex.service.ts

```diff
@@ -4,19 +4,18 @@ import { Codex } from '@codex-data/sdk';
 import moment from 'moment';
 import { Deployment, NATIVE_TOKEN } from '../deployment/deployment.service';
 
-export const SEI_NETWORK_ID = 531;
-export const CELO_NETWORK_ID = 42220;
-
 @Injectable()
 export class CodexService {
   private sdk: Codex;
 
   constructor(private configService: ConfigService) {
     const apiKey = this.configService.get<string>('CODEX_API_KEY');
-    this.sdk = new Codex(apiKey);
+    if (apiKey) {
+      this.sdk = new Codex(apiKey);
```

Comment:
Did you notice that inside the Codex ctor there's a proper check for whether the key is defined - otherwise throwing an exception? Adding this `if` is a significant change to this behavior.

Reply:
Isolated this change into a separate PR, improved it a little bit and explained it there: https://github.com/bancorprotocol/carbon-backend/pull/82

I didn't find any proper check for this key on current version